test/math.jl

```diff
@@ -733,6 +733,7 @@ end
   @test !isapprox(1, 1+1.0e-12, norm=x->1)
 end
 
+#= disabled for now
```

Comment:
Can these be re-enabled? Seems to pass for me.

Reply:
Yeah I'll add them back in